### DISCUSSION TITLE
Enhancement: Accessibility to those with dyslexia, autism and other sight difficulties.

### DISCUSSION THREAD

**Discussion by thecogitator** [FunkinCrew/Funkin#1645] [Ideas]
**I can't see any of the arrows!**

I really want to love this game so much, but for someone with autism and reading difficulties, I'm really struggling to see the notes and therefore can't have any sort of good experience of the game. 
Combined with the arrows blending in with the chaotic background and the constant shakiness and flashes of the background; this game renders itself unfortunately very unfriendly to those with learning disabilities and I'm getting all sorts of headaches from playing it. 
The music is wonderful, but please, provide an option that's a bit more stable, for example have the scene in a frame and then have a plain background for the arrows so that us with dyslexia, autism and other sight difficulties can enjoy this great game. Even having plain white arrows or arrows that stand out more from the background would be the biggest help.
The music and characters are so beautifully created and I wish to be able to play it with my difficulties.

**Comment by kyandora** [comment]
Something like this?
![cleaner background](https://user-images.githubusercontent.com/71771686/116496558-9d79af80-a8a5-11eb-8ed7-ce61319b0164.png)
